fde/core/_scope.py

```diff
@@ -113,7 +114,7 @@ def updateDomain( self, path_dict, keyOp=str.split ):
 
     def iterDomain( self, paths, keyOp = str.split, default = LookupError ):
         keyOp = keyOp or (lambda k: [k])
-        for p in map( keyOp, paths ):
+        for p in list(map( keyOp, paths )):

```

Comment:
Did this `map` really lead to problems in python 3?
Explicit conversion to `list` should not be necessary here.